ownCloudAppShared/User Interface/Card Presentation Controller/CardPresentationController.swift

```diff
@@ -120,6 +126,13 @@ final class CardPresentationController: UIPresentationController, Themeable {
 		super.init(presentedViewController: presentedViewController, presenting: presentingViewController)
 
 		Theme.shared.register(client: self, applyImmediately: true)
+
+		presentedViewController.view.layer.shadowColor = UIColor.black.cgColor
```

Comment:
I believe the shadow will not be visible, because it is also clipped by `layer.masksToBounds = true` further down below. The lines defining the shadow could therefore be removed.